### DISCUSSION TITLE
OneBot V12 候选草案相关

### DISCUSSION THREAD

**Discussion by RinLin-NYA** [howmanybots/onebot#254] [问答] [answered]
现在是2024.03.30，距离 OneBot V12 宣布进入候选阶段并冻结决议案已经过去了一年多时间(
请问 OneBot V12 的候选预计什么时候可以成为正式标准(

**Comment by stdrc** [comment, marked as answer]
摆了（

感觉现在可以认为不会再改 OB12 了吧，已经算是正式了

**Reply by RinLin-NYA** [reply]
有实无名了属于是
